<?xml version="1.0"?>
<launch>
    <!-- Run the yolo_detection_node -->
    <node name="yolo_detection_node" pkg="ros_ml" type="yolo_detection_node.py" output="screen">
        <!-- The YOLO detection input image topic -->
        <param name="yolo_input_image_topic" value="/econ_camera/image_raw/compressed" />

        <!-- Rotation [0, 1, 2, 3] <_> [0, 90, 180, 270] -->
        <param name="rotation" value="2" />

        <!-- The YOLO detection image topic -->
        <param name="yolo_image_topic" value="yolo_detection_image" />

        <!-- The YOLO detection result topic -->
        <param name="yolo_result_topic" value="yolo_detection_result" />
    </node>
</launch>
